src/helpers/utils.tsx

```diff
@@ -94,7 +94,7 @@ export interface GeoJSON {
 export function getLocationColumns(
   locations: LocationItem[] = locationHierarchy,
   padHeader: boolean = false
-): Column[] {
+) {
```

Comment:
If not `Column[]`, what is being returned?  Can we add params and returns to the docstring?

Reply:
ok, this function is being used by 2 versions of React-table with different type declarations for Column, a union intersection will also not suffice as a solution here.

Reply:
Ok great, that makes sense 👍